packages/react-devtools-shared/src/config/DevToolsFeatureFlags.core-oss.js

```diff
@@ -15,7 +15,7 @@
 
 export const enableProfilerChangedHookIndices = false;
 export const isInternalFacebookBuild = false;
-
+export const enableNamedHooksFeature = false;
```

Comment:
why are the values false here?

Reply:
I think because we explicitly don't want it enabled in React Native yet?